src/hardware/powermgm.cpp

```diff
@@ -40,7 +40,6 @@
 #include "sound.h"
 
 #include "gui/mainbar/mainbar.h"
-#include <app/alarm_clock/alarm_in_progress.h>
```

Comment:
I did not find usage of this header file. Thus removing this import will remove dependency on alarm_clock app.

Reply:
This is probably an artifact from a first RTC alarm implementation.